ERCS/erc-7651.md

```diff
@@ -0,0 +1,349 @@
+---
+eip: 7651
+title: Fractionally Represented Non-Fungible Token
+description: A specification for fractionally represented non-fungible tokens.
+author: Acme (@0xacme), Calder (@caldereth)
+discussions-to: https://ethereum-magicians.org/t/erc-7651-fractionally-represented-non-fungible-token/19176
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-03-05
+requires: 20, 165, 721
+---
+
+## Abstract
+
+This proposal introduces a standard for fractionally represented non-fungible tokens, allowing NFTs to be managed and owned fractionally within a single contract. This approach enables [ERC-721](./eip-721.md) NFTs to coexist with [ERC-20](./eip-20.md) tokens seamlessly, enhancing liquidity and access without dividing the NFT itself, and without the need for an explicit conversion step. The standard includes mechanisms for both fractional and whole token transfers, approvals, and event emissions.
```

Comment:
```suggestion
This proposal introduces a standard for fractionally represented non-fungible tokens, allowing NFTs to be managed and owned fractionally within a single contract. This approach enables NFTs to coexist with fungible token seamlessly, enhancing liquidity and access without dividing the NFT itself, and without the need for an explicit conversion step. The standard includes mechanisms for both fractional and whole token transfers, approvals, and event emissions. This project takes inspiration from [ERC-721](./eip-721.md) and [ERC-20](./eip-20.md), but is not fully compatible with either of them.
```

Reply:
Let's ensure it's clear that this ERC doesn't comply with ERC-20 and ERC-721 so readers make more informed decision when adopting

Reply:
Sounds good, updated in the latest commit!